.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         ruby: [3.0, 3.3]
-        node: [16, 20]
+        node: [18, 20]
```

Comment:
As per the discussion [here](https://github.com/shakacode/react_on_rails/pull/1598#discussion_r1461220802), we bump the Node version to 18+ to be able to run our generator.

Reply:
This needs to be reverted after the #1598 merge.